nestkernel/conn_builder.cpp

```diff
@@ -382,9 +382,18 @@ nest::ConnBuilder::change_connected_synaptic_elements( index sgid,
 void
 nest::ConnBuilder::connect()
 {
+  if ( kernel().model_manager.connector_requires_symmetric( synapse_model_ )
+    && not symmetric_ && not is_symmetric() )
```

Comment:
@janhahne It seems to me we get a bit of an inflation here: `symmetric_`, `is_symmetric()` and `supports_symmetric()`. Could we at least integrate `symmetric_` and `is_symmetric()`?